packages/gestalt/src/TableOfContents/TableOfContentsAnchor.js

```diff
@@ -0,0 +1,80 @@
+// @flow strict
+import { type Node } from 'react';
+import classNames from 'classnames';
+import styles from './TableOfContentsAnchor.css';
+import Box from '../Box.js';
+import Colors from '../Colors.css';
+import { useNesting } from '../contexts/NestingProvider.js';
+import Flex from '../Flex.js';
+import Layout from '../Layout.css';
+import TapArea, { type OnTapType } from '../TapArea.js';
+import Text from '../Text.js';
+import useInteractiveStates from '../utils/useInteractiveStates.js';
+
+const NESTING_MARGIN_START_MAP = {
+  '1': '12px',
```

Comment:
can we replace this with --space tokens?

Reply:
i'd like to prevent this map but for this first iteration, we can go ahead.

Reply:
There is no spacing tokens with these values unfortunately

Reply:
Actually, this might be okay for now to not use spacing since it's a `margin-start` value, and it might change. I can revisit this when we get more tokens, because we're may not have the 92px support either